containers/sfdx-project/test-project/test.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
What is this and do we need it?

Reply:
It's something provided by the default template and suggested ( optional ) in the collaboration docs.  But it doesn't get installed by vscode into the user's workspace.  It's in the .npmignore and apparently that will ignore it.

In the test I basically just made sure git, sfdx, and java are installed.

Reply:
I think we can remove it since we do this already as part of https://github.com/forcedotcom/salesforcedx-docker/blob/develop/.circleci/config.yml

No need to duplicate and maintain at two different places since this shell script is unlikely to be run.

Reply:
Good!